### PR TITLE
Enhance Statistics Panel

### PR DESCRIPTION
**Pull request by TechLinkRon**
## Description
Make use of [DataTables](http://datatables.net) plug-in for jQuery to enhance the Pokemon statistics table with sorting capabilities.  Also make some layout and styling changes for consistency within the Stats panel as well as with the Options panel.

## Motivation and Context
Makes the stats panel even MORE useful

## How Has This Been Tested?
Tested on my own setup.  Checked how it responded to having markers turned on and off via the Options panel.

## Screenshots (if appropriate):

sortnameasc

sortnamedesc

sortcountasc

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by CoolCanuck** [comment]
Thanks. Sorting is awesome.

File: static/js/stats.js [hunk 1]
```diff
@@ -23,18 +26,38 @@ function countMarkers () { // eslint-disable-line no-unused-vars
...
+      .clear()
+      .rows.add(pokeCounts)
+      .draw()
   } else {
-    document.getElementById('pokemonList').innerHTML = 'Pokémons markers are disabled'
+    pokeStatTable
+      .clear()
+      .draw()
+
+    document.getElementById('pokeStatStatus').innerHTML = 'Pokemon markers are disabled'
```
Comment (jaquer) [inline static/js/stats.js]: This should be "`Pokémon markers are disabled"

**Comment by TechLinkRon** [comment]
Thanks for the catch. Spelling corrected.

**Comment by averylarry** [comment]
Could you maybe post some screenshots of what this does?

**Comment by CoolCanuck** [comment]
This is still pending? ugh.

**Comment by CT9AJ** [comment]
There is normal Total Count for Pokemon with this. Can that be added back?

**Pull request merged**